template_preprocess/process/handlebars.py

```diff
@@ -1,14 +1,37 @@
 import re
 from django.template import Template, Context
+from django.conf import settings
+
+# The default values for the exempt django template functions/variables
+# that will not be processes when in handlebars functions on the preprocess
+DEFAULT_EXEMPT_HANDLEBARS_VARIABLES = ["csrf_token"]
 
 
 def process(content, seen_templates, template_processor):
     def replace_handlebars(match):
         template_string = ("{% load templatetag_handlebars %}"
                            "{% load static %}"+match.group(0))
+
+        handlebars_variables = getattr(settings,
```

Comment:
good approach, but i'm concerned the exclusion list will bite us when the next tag is added.  would it make sense to replaced with a generalized backreference replacement so we don't need an explicit list?